hack/release-artifacts.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+GOOS=${GOOS:-"linux"}
+GOARCH=${GOARCH:-"amd64"}
+
+VERSION=${VERSION:-"latest"}
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")
+OUT_BASE_PATH=${OUT_BASE_PATH:-"${REPO_ROOT}/out"}
+RELEASE_PATH="${OUT_BASE_PATH}/release-artifacts"
+CHART_PATH="${OUT_BASE_PATH}/charts"
+
+rm -rf "${RELEASE_PATH}"
+mkdir -p "${RELEASE_PATH}"
+
+BINS=("kurator")
```

Comment:
Not sure i understand what does BINS mean with an array

Seems this shoud always be kurator

Reply:
yes, just in case there's another cli binary in the future?